models/team_test.go

```diff
@@ -184,6 +184,62 @@ func TestFindTeams(t *testing.T) {
 	}
 }
 
+// TestTeamById tests TeamById function.
+//
+func TestTeamById(t *testing.T) {
```

Comment:
You are not following the same pattern as TestUserById. In your case, the creation of team is in the test loop.
When you reviewed TestUserById, you told me to move the entity creation outside the test loop...